packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.constant.js

```diff
@@ -5,9 +5,25 @@ export const USER_TYPE = {
   default: 'others',
 };
 
+export const LEGAL_LINK1 =
+  'https://www.cert-in.org.in/PDF/CERT-In_Directions_70B_28.04.2022.pdf';
+export const LEGAL_LINK2 = 'https://www.cert-in.org.in/Directions70B.jsp';
+export const LEGAL_LINK3 = {
+  fr_FR: 'https://www.ovhcloud.com/fr/terms-and-conditions/privacy-policy/',
+  de_DE: 'https://www.ovhcloud.com/de/terms-and-conditions/privacy-policy/',
+  es_ES: 'https://www.ovhcloud.com/es-es/terms-and-conditions/privacy-policy/',
+  it_IT: 'https://www.ovhcloud.com/it/terms-and-conditions/privacy-policy/',
+  pl_PL: 'https://www.ovhcloud.com/pl/terms-and-conditions/privacy-policy/',
+  pt_PT: 'https://www.ovhcloud.com/pt/terms-and-conditions/privacy-policy/',
+  OTHERS: 'https://www.ovhcloud.com/en-gb/terms-and-conditions/privacy-policy/',
+};
```

Comment:
**nitpick:** The links are redirecting to Website which doesn't use Locale but use `ovhSubsidiary`. So, could you please define the links based on sub instead on locale?

Reply:
done Anoop